### ISSUE TITLE
Simulate host node outage (proxmox)

### ISSUE THREAD

**Issue by billimek** [billimek/k8s-templates#6]
* to see how the cluster handles the outage
* to see how the HA things like traefik handle the outage
* to see how pods get properly rescheduled

**Comment by billimek** [comment]
Tested this with just a single node outage.

It seemed to handle this properly but took longer than I expected.  Part of the issue is that I think I was also dealing with a traefik big with websockets and host headers which was confusing results.

Will test some more and eventually try testing with simulating an outage of k8s and k8s-c (all running on proxmox by default) next.

**Comment by billimek** [comment]
Tested again with simulating a 2-node outage by shutting-down `k8s` and `k8s-c`.  It was not pretty:

1. The k8s cluster itself seemed to stop responding to API requests which seems reasonable because etcd did not have quorum and probably did this as a self-preservation action
1. traefik basically stopped responding across all nodes
1. Started downed nodes back up - kubernetes repaired itself automatically and started servicing requests again
1. Traefik never came back online
1. It appears that rook and possibly consul crapped out when they lost 2 out of the three nodes and would not recover
1. Instead of trying to figure out how to properly 'restore' the rook cluster, I had to completely blow it away (including the data drives and `/var/lib/rook` before I could get it back online.  It was ugly.
1. Once rook was back, getting consul and traefik back was like installing them fresh again.
